plugins/occt/f3d-occt-formats.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
-  <mime-type type="application/vnd.step">
+  <mime-type type="model/step">
```

Comment:
While step is part of iana: https://www.iana.org/assignments/media-types/model/step, it is not present in freedesktop yet https://gitlab.freedesktop.org/xdg/shared-mime-info

We should open an issue so it can be added. I did that successfully already for certain formats.